infra/container_apps/onboarding-ms/env/dev/terraform.tfvars

```diff
@@ -10,7 +10,7 @@ tags = {
 }
 
 container_app = {
-  min_replicas = 1
+  min_replicas = 0
```

Comment:
it might be better to have 1 because onboarding-ms takes a long time to startup...

Reply:
It took 15 seconds to cold start, isn't it acceptable for the first call of the day?
![image](https://github.com/pagopa/selfcare-onboarding/assets/7879858/7d9f3d31-0391-48d4-8224-dc97b40c35df)

Reply:
problem could happen with onboarding requests which are expensive......we should decrease cold start in the future working on some aspects inside ms

Reply:
Like what? I'm asking because I'm setting up probes on container app

To be clear, the above `min_replicas` set to `0` is for `dev` and `uat` envs only, `prod` is always warm